src/lib/viewers/BaseViewer.js

```diff
@@ -1037,6 +1040,32 @@ class BaseViewer extends EventEmitter {
         this.annotator.scrollToAnnotation(data);
     }
 
+    /**
+     * Handles retrieving the active annotation id from a deep-linked annotation and scrolls to the annotation.
+     * @param {Object} event - annotations array from emitted from box-annotations.
+     * @param {Array} event.annotations - annotations array from emitted from box-annotations.
+     * @return {void}
+     */
+    handleAnnotationsOnLoad = ({ annotations = [] }) => {
+        const {
+            file: { id },
+        } = this.options;
+
+        const activeAnnotationId = getProp(this.options, `fileOptions.${id}.annotations.activeId`, null);
+
+        if (!activeAnnotationId) {
+            return;
+        }
+
+        const annotation = annotations.find(entry => entry.id === activeAnnotationId);
+
+        if (!annotation) {
```

Comment:
Do these need to be separate guard clauses? If `activeAnnotationId` is undefined, it won't match any results in the `.find` call above.

Reply:
Agreed, but the feedback from Conrad was to add the guard which I didn't really see a problem with.

Reply:
Sure, keep the guard, just replace the `annotationId` check with it.